omod/src/main/webapp/pages/visit/templates/vaccination/vaccinationTable.gsp

```diff
@@ -33,7 +33,7 @@
                 <th>{{ vaccination.label | translate }}</th>
                 <td nowrap ng-repeat="sequence in sequences" ng-class="{ impossible: !isDoseValidForVaccination(sequence, vaccination) }">
                     <span ng-show="existingDose(sequence, vaccination)" ng-class="{'bold': existingDoseInEncounter(sequence, vaccination)}">
-                        {{ existingDose(sequence, vaccination) | groupMember:Concepts.vaccinationDate | obs:"value" | serverDate : 'dd-MMM-yyyy' }}
+                        {{ existingDose(sequence, vaccination) | groupMember:Concepts.vaccinationDate | obsValueDateLocalized }}
                         <a class="delete-action" ng-show="canDelete(sequence, vaccination)" ng-click="confirmDelete(sequence, vaccination)"><i class="icon-remove"></i></a>
```

Comment:
I could not figure out how to get the DatetimeFormats.dateLocalized constant from constants.js to be accessible from within the vaccinationTable gsp, but since the filters seem to be visible, and there is a pattern for using the constants in the filters that works, I just created a new filter to do what I needed to do, and then used the  filter here.